tensorflow-core/tensorflow-core-api/configure

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+#
+# /* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Unnecessary `/*`

Reply:
And a `*/` at the end of the license block